tests/ssl-opt.sh

```diff
@@ -3739,106 +3739,55 @@ run_test    "Large packet TLS 1.2 AEAD shorter tag" \
             -c "16384 bytes written in 1 fragments" \
             -s "Read from client: 16384 bytes read"
 
-# Tests for DTLS HelloVerifyRequest
+# Tests for ECC extensions (rfc 4492)
```

Comment:
Shouldn't this come before the DTLS tests, rather than replacing them?

Reply:
I think it does come before the DTLS tests. I think this is just a diff alignment issue, isn't it?

Reply:
Yes, I believe it was a merge conflict, that these lines were accidentally deleted. I will return them, and put them after line  3792 which comments on DTLS tests